src/main/kotlin/com/cognifide/gradle/aem/environment/docker/runtime/Desktop.kt

```diff
@@ -17,6 +17,19 @@ class Desktop(aem: AemExtension) : Base(aem) {
 
     override fun determinePath(path: String) = Formats.normalizePath(path)
 
+    override val hostInternalIp: String?
+        get() = when {
+            OperatingSystem.current().isWindows || OperatingSystem.current().isMacOsX -> null
+            else -> detectHostInternalIp() ?: aem.props.string("environment.docker.desktop.hostInternalIp") ?: "172.17.0.1"
```

Comment:
this IP sometimes might change, we need to implement a way for determining it automatically; however this fallback worked on my clean Ubuntu with just installed Docker

Reply:
`docker run -it alpine /bin/ash -c "ip -4 route list match 0/0" | awk '{ print $3 }'`

returns IP on ubuntu